components/zoho_crm/zoho_crm.app.mjs

```diff
@@ -1,12 +1,74 @@
-const axios = require("axios");
+import { axios } from "@pipedream/platform";
 
-module.exports = {
+export default {
   type: "app",
   app: "zoho_crm",
-  methods: {
-    _authToken() {
-      return this.$auth.oauth_access_token;
```

Comment:
Was this method (`_authToken`) intended to be removed? It looks like it's still used in `_makeRequestConfig()`.